javascript/modules/client/src/internal/interfaces/dao.ts

```diff
@@ -120,7 +123,7 @@ export interface ICreateProposal {
 
 export interface IProposalAction {
   to: string;
-  value: number;
+  value: bigint;
```

Comment:
Might we accept `number | bigint` and then make sure to always convert to bigint?
"No" is a great answer too

```suggestion
  value: number | bigint;
```